src/styles/card/card.scss

```diff
@@ -0,0 +1,136 @@
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: $field-size;
+  margin: $padding-side auto;
+  min-width: 1024px;
+  margin-top: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-size;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: $cards-gap;
+  margin-bottom: 46px;
+  padding: 32px 16px 16px;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__description {
+    display: flex;
+    align-items: flex-start;
+    margin-bottom: 4px;
+    margin-top: 0;
+    color: $link-color;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-number {
+    display: flex;
+    text-justify: initial;
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+  }
+
+  &__reviews {
+    display: flex;
+    align-self: flex-start;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    height: 16px;
+    width: 100%;
+
+    &-count {
+      font-size: 10px;
+      text-align: center;
+      line-height: 14px;
+      color: $link-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    line-height: 18px;
+    margin-bottom: 16px;
+
+    &-title {
+      margin: 0;
+      color: #616070;
+      font-weight: 400;
+      font-size: 12px;
+    }
+
+    &-amount {
+      color: $link-color;
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    border: 1px solid transparent;
+    padding: 11px 0;
+    background: #00acdc;
+    border-radius: 5px;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $primary-color;
+      background-color: #fff;
+      border-color: #00acdc;
+    }
+  }
+}
+
+.stars {
```

Comment:
Create a separate file for stars block
![image](https://user-images.githubusercontent.com/96409563/225566868-a9967712-15a7-4cf1-8c55-42dbb455bdec.png)